clustered/osgi-test/src/test/java/org/ehcache/osgi/OsgiTestUtils.java

```diff
@@ -155,7 +114,16 @@ public static Cluster startServer(Path serverDirectory) throws IOException {
       "-Dtc.install-root=" + serverDir,
       "-cp", serverDir.resolve("lib").resolve("tc.jar").toString(),
       "com.tc.server.TCServerMain",
-      "-f", configFile.toString()));
+      "--cluster-name=foo",
```

Comment:
Note: cluster-name required to pre-activate a node at startup

Reply:
Could this startup code lean on the new config file for most of its defaults?

Reply:
oh yes we could point to it now!

Reply:
wait! no!
default config file:

```
stripe.1.node.1.node-group-port=9430
stripe.1.node.1.node-port=9410
stripe.1.node.1.node-log-dir=%H/terracotta/logs
```
whereas the CLI here need to point to different generated values.

`--config-file` and CLI parameters are mutually exclusive.

Reply:
Wouldn't it be lovely if the CLI could override settings in the config file? ;)

Reply:
it is not consistent because it means that you could have 2 nodes that are meant to be activated in the same stripe that are activated and started with the same config, but if one of them is overriding the config passed, then they won't have the same topology and it will fail.

When using config a file, it means the user sets a cluster configuration to be used across th node and this config must not be altered. All the nodes must have the same.

When starting with the CLI, you are not configuring a topology, you are just creating a 1-node-cluster based on the CLI parameters.

This is not the same use case.